templates/README.md

```diff
@@ -0,0 +1,28 @@
+# Using templates to implement Secretless Connector Plugins
```

Comment:
maybe this content should be moved to `examples/connector_templates` instead of putting it in `templates`? I'm basing this suggestion on the [go project layout standards](https://github.com/golang-standards/project-layout#other-directories)

Reply:
do you mean move the whole directory?